fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/election/CastVote.java

```diff
@@ -1,16 +1,16 @@
 package com.github.dedis.popstellar.model.network.method.message.data.election;
 
+
 import com.github.dedis.popstellar.model.network.method.message.data.Action;
 import com.github.dedis.popstellar.model.network.method.message.data.Data;
 import com.github.dedis.popstellar.model.network.method.message.data.Objects;
 import com.google.gson.annotations.SerializedName;
 
 import java.time.Instant;
 import java.util.Arrays;
-import java.util.Collections;
 import java.util.List;
 
-public class CastVote extends Data {
+public class CastVote <E> extends Data {
 
```

Comment:
What does exactly this <E> mean ?

Reply:
It's a generic type, CastVote can be either encrypted or not. Type is specified when the class is instantiated